firefox-ios/Client/Frontend/Autofill/Address/AddressListViewModel.swift

```diff
@@ -0,0 +1,64 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+import Common
+import Shared
+import Storage
+
+// TODO: PHASE-2 FXIOS-7653
+// AddressListViewModelDelegate: A protocol to notify delegates about address updates.
+// protocol AddressListViewModelDelegate: AnyObject {
+//     func didUpdateAddresses(_ addresses: [Address])
+// }
+
+extension Address {
+    var fullName: String {
+        return "\(givenName) \(familyName)"
+    }
+
+    var addressCityStateZipcode: String {
+        return "\(addressLevel2),  \(addressLevel1) \(postalCode)"
```

Comment:
Should there really be two spaces after the comma?

Reply:
Removed space